package.json

```diff
@@ -7,40 +7,40 @@
     "url": "https://github.com/mozilla/chronicle/issues"
   },
   "dependencies": {
-    "bell": "2.0.0",
+    "bell": "2.1.0",
     "boom": "2.6.1",
     "commander": "2.6.0",
-    "convict": "0.6.0",
-    "elasticsearch": "3.1.1",
-    "embedly": "1.0.2",
-    "hapi": "8.1.0",
+    "convict": "0.6.1",
+    "elasticsearch": "3.1.3",
+    "embedly": "1.0.4",
+    "hapi": "8.2.0",
     "hapi-auth-cookie": "2.0.0",
     "joi": "5.1.0",
-    "mozlog": "1.0.1",
-    "node-resque": "0.11.5",
+    "mozlog": "1.0.2",
+    "node-resque": "0.11.6",
```

Comment:
node-resque 0.11.5 -> 0.11.6:
- some changes to event loop lag detection. since event loop lag is used to decide how many workers to spawn, this could have significant impact if we had serious load. luckily, we don't ;-) https://github.com/taskrabbit/node-resque/commit/8770bbe